test/parallel/test-crypto-cipher-decipher.js

```diff
@@ -113,3 +113,27 @@ testCipher2(Buffer.from('0123456789abcdef'));
   c.update('update', 'utf-8');
   c.final('utf8');  // Should not throw.
 }
+
+//#8236 regression tests, 'ucs2', 'usc-2', 'utf-16le', 'utf16le' are identical.
```

Comment:
nit: Using the full github URL for issue references is cool because that’s more accessible (even clickable in many cases) and helps distinguish from the https://github.com/nodejs/node-v0.x-archive issue tracker